test/verify/conftest.py

```diff
@@ -33,9 +52,10 @@ def pytest_collection_modifyitems(session: pytest.Session, items: List[pytest.It
     assert isinstance(items, list)
 
     def is_nondestructive(item: pytest.Item) -> bool:
+        attr = '_testlib__nondestructive'
         assert isinstance(item, pytest.Function)
         assert isinstance(item.parent, pytest.Class | pytest.Module)
-        return testlib.get_decorator(item.obj, item.parent.obj, "nondestructive", False)
+        return getattr(item.obj, attr, getattr(item.parent.obj, attr, False))
```

Comment:
Grumbling a bit about the code and naming convention duplication..  Why can't this import testlib after setting sys.path?